.github/workflows/ci-tests.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         Set-Content -Path certificate/tempCert.txt -Value $env:WINDOWS_PFX
         certutil -decode certificate/tempCert.txt certificate/certificate.pfx
         Remove-Item –path certificate -include tempCert.txt
-        Import-PfxCertificate -FilePath certificate/certificate.pfx -CertStoreLocation Cert:\LocalMachine\My -Password (ConvertTo-SecureString -String $env:WINDOWS_PFX_PASSWORD -Force -AsPlainText)
+        Import-PfxCertificate -FilePath certificate/certificate.pfx -CertStoreLocation Cert:\LocalMachine\My -Password (ConvertTo-SecureString -String $WINDOWS_PFX_PASSWORD -Force -AsPlainText)
```

Comment:
we just tried with Justin. This command did not work on my machine. The error was the referenced string being null. After removing `env:` part, it worked 🤷

Reply:
So I tried to use the string that is being referenced by `WINDOWS_PFX_PASSWORD' directly into this command. And it worked. So it was obviously something related to `$env:WINDOWS_PFX_PASSWORD` not being able to resolve.

Reply:
It is only `null` after changes in this commit, in my commit it wasn't `null` and worked fine, in fact the whole app was built, just signature at the very end didn't work. There was no point of changing this variable usage since there was no error related to it.

Reply:
True. For the CI, it is not needed, and now the CI is creating the same error happened on my machine with the "env:" version. 
However, isnt it weird that it is behaving differently in here? Maybe because of windows-latest, my machine is windows10

Reply:
Not sure what error you mean exactly, but please roll-back this change.

windows-latest is Windows Server 2019 at the moment on GitHub Actions, which is similar to Windows 10. Windows Server 2022 is not the default **yet**.